tests/__init__.py

```diff
@@ -32,9 +32,44 @@ def test_ids_for_address(self):
         self.assertEqual(ids, ['America/Los_Angeles'])
         ids = tztrout.tz_ids_for_address('US', state='CA')
         self.assertEqual(ids, ['America/Los_Angeles'])
+        ids = tztrout.tz_ids_for_address('US', state='CA', city='')
+        self.assertEqual(ids, ['America/Los_Angeles'])
+        ids = tztrout.tz_ids_for_address('US', state='CA', city='Palo Alto')
+        self.assertEqual(ids, ['America/Los_Angeles'])
+        ids = tztrout.tz_ids_for_address('US', state='', city='Palo Alto')
+        self.assertEqual(ids, ['America/Los_Angeles'])
+        ids = tztrout.tz_ids_for_address('US', city='Palo Alto')
+        self.assertEqual(ids, ['America/Los_Angeles'])
+        ids = tztrout.tz_ids_for_address('US', state='', city='Palo Alto')
```

Comment:
Isn't this the same as the test 4 lines above?

Reply:
I guess I wanted to have a `'', ''` case here...